### DISCUSSION TITLE
Using gpt-3.5-turbo to format the result from Code Execution to reduce cost.

### DISCUSSION THREAD

**Discussion by myfunc** [microsoft/autogen#439] [Ideas]
Use case:
I use the default gpt-4 model, and when User Proxy executes the code, it sends the entire result to chat. Sometimes this can be more than 6k tokens with useless information. 
Performing a simple task can cost a few dollars because of this.

To analyze the long result I see a solution:
- Use gpt-3.5-turbo to analyze the output to the console and make a short summary with key points, then pass it to the Coder agent.
This would reduce the cost a lot. 

![image](https://github.com/microsoft/autogen/assets/19294627/daff8a04-b99e-4487-91e9-367083a99e01)

**Comment by sonichi** [comment]
Clever! You may be interested in #131 
@kevin666aa for awareness.
